src/content/components/FeatureView/FeatureView.js

```diff
@@ -310,121 +426,193 @@ export class FeatureView extends React.PureComponent {
+        hideIfEmpty={true}
+        crossOutResolved={false}
+        bugs={bugs.resolved}
+        columns={[...resolvedDisplayColumns, "target_milestone"]}
+      />
+      <p>
+        <button className={gStyles.primaryButton} onClick={bulkEditAll}>
+          Edit all in Bugzilla
+        </button>
+      </p>
+      <MiniLoader hidden={!awaitingNetwork} />
+    </React.Fragment>
+  ) : (
+    <Loader />
+  );
+};
 
-  componentWillMount() {
-    this._isMounted = true;
-    this.currentRelease = this.context.iterations
-      .getIteration()
-      .number.split(".")[0];
-    this.prevRelease = parseInt(this.currentRelease, 10) - 1;
-    this.nextRelease = parseInt(this.currentRelease, 10) + 1;
-    const upliftTrackingField = `cf_tracking_firefox${this.prevRelease}`;
-    this.allColumns = displayColumns.concat([
-      "target_milestone",
-      "status",
-      "resolution",
-      "last_change_time",
-      "whiteboard",
-      "keywords",
-      "flags",
-      "blocks",
-      "type",
-      "cf_fx_points",
-      upliftTrackingField,
-      `cf_status_firefox${this.prevRelease}`,
-      `cf_status_firefox${this.currentRelease}`,
-      "cf_last_resolved",
-    ]);
-    this.getBugs(this.props.match.params.id);
-  }
+export const FeatureView = ({
+  match: {
+    url,
+    params: { id },
+  },
+}) => {
+  const { metas, iterations } = useContext(GlobalContext);
 
-  componentWillUnmount() {
-    this._isMounted = false;
-  }
+  const metaId = Number(id);
+  const { displayName: metaDisplayName, component } = metas.find(
+    meta => meta.id === metaId
+  );
 
-  render() {
-    const metaId = Number(this.props.match.params.id);
-    const { displayName: metaDisplayName, component } = this.context.metas.find(
-      meta => meta.id === metaId
-    );
-    const bugsByRelease = this.sortByRelease(
-      this.state.bugs,
-      this.state.subMetas
-    );
+  const currentRelease = useMemo(
+    () => iterations.getIteration().number.split(".")[0],
+    [iterations]
+  );
+  const prevRelease = useMemo(
+    () => parseInt(currentRelease, 10) - 1,
+    [currentRelease]
+  );
+  const nextRelease = useMemo(
+    () => parseInt(currentRelease, 10) + 1,
+    [currentRelease]
+  );
+  const upliftTrackingField = useMemo(
+    () => `cf_tracking_firefox${prevRelease}`,
+    [prevRelease]
+  );
+  const includeFields = useMemo(
+    () =>
+      displayColumns.concat([
+        "target_milestone",
+        "status",
+        "resolution",
+        "last_change_time",
+        "whiteboard",
+        "keywords",
+        "flags",
+        "blocks",
+        "type",
+        "cf_fx_points",
+        upliftTrackingField,
+        `cf_status_firefox${prevRelease}`,
+        `cf_status_firefox${currentRelease}`,
+        "cf_last_resolved",
+      ]),
+    [currentRelease, prevRelease, upliftTrackingField]
+  );
 
-    return (
-      <Container
-        loaded={this.state.loaded}
-        heading={
+  return (
+    <Container
+      loaded={true}
```

Comment:
Noticed Remaining/Open View doesn't refresh when switching between features and above loaded state needs to maintained to force refresh

Reply:
Good catch, I forgot to add `key={metaId}`